assets/dev/js/frontend/handlers/base-swiper.js

```diff
@@ -11,22 +11,13 @@ export default class SwiperHandlerBase extends BaseHandler {
 		return this.elements.$slides.length;
 	}
 
-	hasThumbs() {
-		return false;
-	}
-
 	// This method live-handles the 'Pause On Hover' control's value being changed in the Editor Panel
 	togglePauseOnHover( toggleOn ) {
 		// These variables exist because different Swiper-based widgets have different container variable names,
 		// E.g. The Media Carousel widget has two Swiper instances in Slideshow mode
-		let swiperInstance = this.swiper,
+		const swiperInstance = this.swiper,
```

Comment:
Now you can completely get rid of these variables, as well as the comments above

Reply:
Done.